src/Progress/style.css

```diff
@@ -0,0 +1,158 @@
+@import '../variables.css';
+@import 'antd/lib/progress/style/index.css';
+
+.ant-progress {
+  @apply font-inter;
+}
+
+/* center line success/exception icons */
+.ant-progress {
+  &.ant-progress-line {
+    & .ant-progress-text {
+      @apply relative;
+    }
+
+    &.ant-progress-default {
+      & .ant-progress-text {
+        &::before {
+          position: absolute;
+          transform: translateY(-40%);
+        }
+      }
+    }
+    &.ant-progress-small {
+      & .ant-progress-text {
+        &::before {
+          position: absolute;
+          transform: translateY(-46%);
+        }
+      }
+    }
+  }
+}
+
+.ant-progress {
+  &.ant-progress-status-success.ant-progress-default {
+    &.ant-progress-circle {
+      & .ant-progress-text {
```

Comment:
Is it possible to reduce the number of selectors? If yes, please do the same thing for other selector too.

Reply:
The original selector is specific. So no. I can change it to one line but it would be difficult to modify